wit-0.3.0-draft/types.wit

```diff
@@ -132,11 +132,11 @@ interface types {
     immutable,
   }
 
-  /// Field keys are always strings.
+  /// Field names are always strings.
   ///
   /// Field keys should always be treated as case insensitive by the `fields`
```

Comment:
```suggestion
  /// Field names should always be treated as case insensitive by the `fields`
```

Reply:
Maybe do another grep -i for "key"; I see a few other "key"s hanging out in comments.